### ISSUE TITLE
Semver-select error 

### ISSUE THREAD

**Issue by cameronelliott** [WillAbides/setup-go-faster#5]
Can this action be used for beta and RC versions?

```Run WillAbides/setup-go-faster@v1.3.0
  with:
    go-version: 1.16rc-1
semver-select: error: improper constraint: 1.16rc-1
semver-select: error: improper constraint: 1.16rc-1
semver-select: error: improper constraint: 
Error: Process completed with exit code 1.
```

**Comment by WillAbides** [comment]
Thanks for reporting this.

There are two things going on here. 

First, it should be `1.16rc1` (without the dash).  But even with the correct version, you would have had a similar error because I introduced a bug in v1.3.0.  For now using `WillAbides/setup-go-faster@v1.2.0` will work.

I'll look at this now and let you know when I have a fix.

**Comment by WillAbides** [comment]
@cameronelliott thanks again for reporting this.  The fix is available in `@v1.3.1`.  Both `WillAbides/setup-go-faster@v1.3.1` and `WillAbides/setup-go-faster@v1` should work now.
